index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <h1 id="title">freeCodeCamp Survey Form</h1>
+    <p id="description">Thank you for taking the time to help us improve the platform</p>
```

Comment:
Classes ( or you can select by other ways) Must be used instead of ID. if you want to uniquely identify an element on that case you can you use id.

Reply:
Kindly consider phrasing your comments more politely, as suggestions instead of orders. Words like `must`, `should` etc sounds like a command. 
The suggestion here is not clear. I am not sure it is wrong to  use an id here. Kindly consider making it more understandable. You can also consider giving clear reasons as to why it should be done that way so that it will be easier for the student to understand and implement. :+1: